packages/aws-cdk-lib/aws-eks/lib/cluster.ts

```diff
@@ -1987,6 +1995,28 @@ export class Cluster extends ClusterBase {
     return this._neuronDevicePlugin;
   }
 
+  /**
+   * Installs the Neuron device plugin RBAC on the cluster if it's not
+   * already added.
+   */
+  private addNeuronDevicePluginRbac() {
```

Comment:
If I use [parseAllDocuments](https://eemeli.org/yaml/#parsing-documents), I don't need to divide `k8s-neuron-device-plugin-rbac.yml` into three files but the return type of `parseAllDocuments` is not equal to the return type of [parse](https://eemeli.org/yaml/#yaml-parse) so `addManifest` function cannot handle parsed yaml.
I think divide `k8s-neuron-device-plugin-rbac.yml` into three files and use `parse` is the simplest solution.